libs/go/libp2p_node/acn/utils.go

```diff
@@ -0,0 +1,136 @@
+/* -*- coding: utf-8 -*-
```

Comment:
Off-topic: I don't think this is relevant in Go source code

Reply:
agree